### PR TITLE
DOCS-5806 Update beta info for trace queries

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?

Changes beta statement, adds to the left nav

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by marcus-crane** [comment]
Oooh, this looks fun! Looking forward to the announcement tomorrow. Nice choice of subtle Github label too. I won't tell anyone 😉

File: config/_default/menus/menus.en.yaml [hunk 1]
```diff
@@ -1806,65 +1806,69 @@ main:
...
+    weight: 605
   - name: Request Flow Map
     url: tracing/trace_explorer/request_flow_map/
     parent: trace_explorer
-    weight: 506
+    weight: 606
+  - name: Trace Queries
+    url: tracing/trace_queries/
+    parent: tracing
+    weight: 7
```
Comment (kayayarai) [inline config/_default/menus/menus.en.yaml:1836]:
```suggestion
```

File: content/en/tracing/trace_queries/_index.md [hunk 2]
```diff
@@ -14,7 +13,7 @@ further_reading:
 ---
 
 {{< beta-callout url="#" btn_hidden="true">}}
-Trace Queries is in private beta. Spans used for queries are from a <a href="#the-data-that-trace-queries-are-based-on"><strong>uniform 1% sample</strong> of ingested spans</a>, not from your existing retention filters.
+Trace Queries is in beta. Spans used for queries are from a <a href="#the-data-that-trace-queries-are-based-on"><strong>uniform 1% sample</strong> of ingested spans</a>, not from your existing retention filters.
```
Comment (kayayarai) [inline content/en/tracing/trace_queries/_index.md:16]:
```suggestion
To join the Trace Queries private beta <a href="/help/">contact Support</a>. Spans used for queries are from a <a href="#the-data-that-trace-queries-are-based-on"><strong>uniform 1% sample</strong> of ingested spans</a>, not from your existing retention filters.
```

**Review by brett0000FF — APPROVED** [review]
LGTM!

**Pull request merged**